plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -322,16 +316,52 @@ private static <K, V> V get(Cache<K, V> cache, K key, Supplier<V> loader)
         return uncheckedCacheGet(cache, key, loader);
     }
 
-    private static <K, V> Map<K, V> getAll(LoadingCache<K, V> cache, Iterable<K> keys)
+    private static <K, V> V getSingle(Cache<K, AtomicReference<V>> cache, K key, Supplier<V> loader)
```

Comment:
I guess the reason why it's not called just `get` is that it "conflicts" with `get(Cache<K, V> cache, K key, Supplier<V> loader)` defined above.
Both get a single entry, so both could be named "get" or "getSingle".

Let's have a name that actually hints at the difference between the methods.

Maybe sth like

```java
/**
 * Gets a single entry from a bulk cache using {@link AtomicReference} as value placeholders.
 */
private static <K, V> V getWithValueHolder(Cache<K, AtomicReference<V>> cache, K key, Supplier<V> loader)
```